### ISSUE TITLE
PostgreSQL: error with first selecting array temp table

### ISSUE THREAD

**Issue by ems2811** [dbeaver/dbeaver#11362]
#### System information: 
- Windows 7 Pro x64
- DBeaver 7.3.5.202102141704
- No additional extensions

#### Connection specification:
- PostgreSQL 9.5
- Standard JDBC PostgreSQL driver
- No tunnels or proxies 

#### Describe the problem you're observing:
Error "Can't resolve data type _int4" at integer array field in temp table if you newer select arrays from other real tables in current connection.

#### Steps to reproduce, if exist:
Open new connection. Execute query below
```
create temp table tt as
select array[1];

select *
from tt;
```
![изображение](https://user-images.githubusercontent.com/22142845/108041107-2f1ba400-704f-11eb-8d82-03f250c666ea.png)

if after this query execute `select array[1];` and again execute query `select * from tt;` - there is no error;

#### Include any warning/errors/backtraces from the logs
```
org.jkiss.dbeaver.model.exec.DBCException: Can't resolve data type _int4
	at org.jkiss.dbeaver.ext.postgresql.model.data.PostgreArrayValueHandler.getValueFromObject(PostgreArrayValueHandler.java:63)
	at org.jkiss.dbeaver.ext.postgresql.model.data.PostgreArrayValueHandler.getValueFromObject(PostgreArrayValueHandler.java:1)
	at org.jkiss.dbeaver.model.impl.jdbc.data.handlers.JDBCComplexValueHandler.fetchColumnValue(JDBCComplexValueHandler.java:50)
	at org.jkiss.dbeaver.ext.postgresql.model.data.PostgreArrayValueHandler.fetchColumnValue(PostgreArrayValueHandler.java:52)
	at org.jkiss.dbeaver.model.impl.jdbc.data.handlers.JDBCAbstractValueHandler.fetchValueObject(JDBCAbstractValueHandler.java:49)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetDataReceiver.fetchRow(ResultSetDataReceiver.java:125)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.fetchQueryData(SQLQueryJob.java:717)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:540)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$0(SQLQueryJob.java:440)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:168)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:427)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:812)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3226)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:118)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:168)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:116)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:4516)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

```

**Comment by uslss** [comment]
thanks for the report

**Comment by LonwoLonwo** [comment]
Fixed in 21.0.0
Will be available [here](https://dbeaver.io/) on 2021-03-01

**Comment by uslss** [comment]
verified
